### DISCUSSION TITLE
Reaching out to mentors of Project 2 (Inference of Neural model in Node.js environment using OpenVINO)

### DISCUSSION THREAD

**Discussion by Huanli-Gong** [openvinotoolkit/openvino#23453] [Google Summer of Code]
Hi, I am Huanli Gong, a master student at Duke University, major in Computer Science. I am looking forward to contribute to and get hands-on experience with open source communities like OpenVINO. The project I'm interested in is 2. Inference of Neural model in Node.js environment using OpenVINO. I want to reaching out to mentors and I have some questions about how to get started: 1. Do I need to complete the prerequisite task before I start? If so, what is the deadline for it? 2. I noticed that there is Application Template provided, then how should I submit it?

**Comment by adrianboguszewski** [comment]
Hi Huanli,

1. The prerequisite task should be completed before you submit your final application
2. It should be submitted through GSoC webapp (https://summerofcode.withgoogle.com/). The submission period starts on 18th March

@vishniakov-nikolai and @almilosz to answer the project's questions

**Reply by Huanli-Gong** [reply]
Thank you for response! I have a few other questions: if I don't get selected for the project I want, is there a chance that I will be assigned to other OpenVINO GSoC project that no one take? and can I submit more than one proposals to OpenVINO as alternatives?

**Reply by adrianboguszewski** [reply]
According to GSoC rules, you can select up to 3 projects as alternatives. You won't be assigned to a random not-selected project by default if someone else will be assigned to your selection.
